test/components/previews/events/rubygem_event/version/pushed_component_preview.rb

```diff
@@ -1,19 +1,12 @@
 class Events::RubygemEvent::Version::PushedComponentPreview < Lookbook::Preview
-  def default(rubygem: Rubygem.first!, # rubocop:disable Metrics/ParameterLists
-    number: "1.0.0", platform: "ruby",
-    version_gid: rubygem.versions.where(number:, platform:).first&.to_gid,
-    pushed_by: "Pusher", actor_gid: version_gid&.find&.pusher&.to_gid)
```

Comment:
Safe navigation rubocop complained about this chain, so I extracted this into the body in each of the 3 previews so it could pass.